### DISCUSSION TITLE
Give developers the possibility to develop plugin

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2058] [Ideas]
It would be awesome like bubble, adalo or as plan to do directus to open a market place where developper can sell their plugins.

Thanks

**Comment by GuilhermeHirsch** [comment]
I agree, @oswaldB. In my opinion, this would be very interesting in automations.

**Comment by oswaldB** [comment]
Yes. But also in elements or component:

+ We can imagine easily imagine :
+ Leaflet
+ Stripe checkout form
+  Captcha
+ ....

**Reply by GuilhermeHirsch** [reply]
We need to consider that Budibase focuses on internal tools. I think the integration with Stripe doesn't fit.

**Comment by oswaldB** [comment]
I am developing a tool to help SME to deliver only when they are payed.

So I am using stripe (thanks to n8n) to help them to get payed on time.

Payment is part of business daily life :)

**Reply by GuilhermeHirsch** [reply]
True, it makes a lot of sense.

**Comment by mjashanks** [comment]
These things will come. I agree that this will be very important to be able to complete the last mile of many applications. Right now, we are still getting some of the fundamentals correct - as you both have experienced!
